pipeline/container/frontend/src/components/play/chat-app/ChatApp.tsx

```diff
@@ -51,32 +51,28 @@ export default function ChatApp({ pipeline }: ChatAppProps): JSX.Element {
   // Constants
   const LS_KEY = `chat-app-prompt-history-${pipeline.image}`;
 
-  const pipelineInputDictIOVariable = pipeline.input_variables.filter(
-    (input) => input.run_io_type === "dictionary"
-  );
-
   // State
   const [inputValue, setInputValue] = useState<string>("");
   const [showSettings, setShowSettings] = useState<boolean>(false);
   const [chatSettings, setChatSettings] = useState<{ [key: string]: any }>();
   const [chats, setChats] = useState<ChatMessages[]>([]);
   const [promptHistory, setPrompHistory] = useState<Prompt[]>();
 
-  const dynamicFields = useMemo(
-    () => generateDynamicFieldsFromIOVariables(pipeline.input_variables),
-    [pipeline.input_variables]
+  // We get a list of input variables that are of type dictionary,
+  // We assume this is the settings for the chat
+  const pipelineInputDictIOVariable = pipeline.input_variables.filter(
+    (input) => input.run_io_type === "dictionary"
   );
+
```

Comment:
I guess if they have multiple input dicts, then I guess this may no longer work. But we can look into that in the future if needs be.

Reply:
yea, that's the thing... I could check for the dict that has a key of "default_system_prompt" but that's just another assumption. Open to suggestions!

Reply:
Ah yeah OK, future selves problem me thinks :p